src/main/resources/application.yaml

```diff
@@ -182,6 +186,7 @@ security:
     caseworker-ia-iacjudge:
       - "generateDecisionAndReasons"
       - "sendDecisionAndReasons"
+      - "applyForFTPAAppellant"
```

Comment:
I think only legal rep should have possibility to apply for FTPL and Home Office within next ticket. What do you think?

Reply:
Thanks @makowalski This has been fixed now.